node/src/Unison/Runtime/ExtraBuiltins.hs

```diff
@@ -81,5 +98,29 @@ makeAPI blockStore crypto = do
              g k v store = pure $ Term.ref r `Term.app` k `Term.app` v `Term.app` store
            op _ = fail "Index.insert unpossible"
        in (r, Just (I.Primop 3 op), unsafeParseType "String -> String -> Store String String -> Remote Unit", prefix "insert")
+     , let r = R.Builtin "Html.getLinks"
+           op [html] = do
+             html' <- whnf html
+             pure $ case html' of
+               Term.Text' h -> Term.vector' . Vector.fromList . map linkToTerm
+                 $ Html.getLinks h
+               x -> Term.ref r `Term.app` x
+           op _ = fail "Html.getLinks unpossible"
+       in (r, Just (I.Primop 1 op), unsafeParseType "String -> Vector Link", prefix "getLinks")
```

Comment:
Should be Text not String in the type. String is not a Unison type. If parser is parsing String as Text that should be updated.